modules/ding_place2book/ding_place2book.module

```diff
@@ -584,10 +532,9 @@ function ding_place2book_event_op($op, $node, $place2book_id = NULL) {
 
       // Catch update error by checking the return code.
       if ($req_result->status_message != 'Accepted') {
-        // @todo: This error handling is not appropriate for normal editors.
-        // Replace with drupal_set_message + watchdog.
         $p2b_error = $req_result->headers['status'] . ' - ' . $req_result->status_message;
         drupal_set_message(t('UPDATE was NOT PERFORMED on Place2book. Place2book returned this error:') . $p2b_error, 'error', FALSE);
+        watchdog('error', 'DELETE was NOT PERFORMED on Place2book. Place2book returned this error: @p2b_error', array('@p2b_error' => $p2b_error), WATCHDOG_ERROR);
```

Comment:
The message sent to watchdog does not correspond to the message displayed to the user. One mentioned `UPDATE`. The other mentions `DELETE`. Judging by the `$op` it seems as if `UPDATE` is right.